theano/tensor/basic.py

```diff
@@ -3263,6 +3263,24 @@ def batched_tensordot(x, y, axes=2):
     return result
 
 
+def cross(x, y):
+    """
+    Warning: cross product for vectors of shape (3,) only
+    :param x: Tensor
+    :param y: Tensor
+    This function computes the cross product between the two tensors
+    using a scalar product with the 3D Levi-Civita symbol
+
+    Returns a tensor 
```

Comment:
Can you tell the output shape at the same time. As David wrote, it will need tests in the file tensor/tests/test_basic.py. In the tests, compare the output again numpy.cross.

thanks